src/types/item/Foraged.svelte

```diff
@@ -14,7 +14,7 @@ const forageGroups = {
   Summer: "forage_summer",
   Autumn: "forage_autumn",
   Winter: "forage_winter",
-  "any season": "trash_forest",
+  "any season": "SUS_trash_no_manmade",
```

Comment:
Maybe check if the group exists?

Reply:
nah, i expect the proper solution would be something like
if (version before 79848){
"any season": "trash_forest",
} else {
"any season": "SUS_trash_no_manmade",
}